R/data.R

```diff
@@ -1,56 +1,5 @@
+#' Sample dataset obtained from the replication archive for Freyaldenhoven et al. (2021)
 #'
 #' @source Dataset in .dta format can be found in the .zip archive in \url{https://data.nber.org/data-appendix/w29170/}
-"df_sample_dynamic"
-
-#' Sample dataset obtained from the replication archive for [Freyaldenhoven et al. (2021)](https://www.nber.org/system/files/working_papers/w29170/w29170.pdf).
-#'
-#' @source Dataset in .dta format can be found in the .zip archive in \url{https://data.nber.org/data-appendix/w29170/}
-"df_sample_static"
-
-#' Sample dataset used in the Examples section in the EventStudyOLS documentation
-#'
-#' @format A data frame with 2000 rows and 13 variables (lead and lags created using post = 1, pre = 1, overidpre = 2, overidpost = 2):
-#' \describe{
-#'   \item{id}{variable to identify units}
-#'   \item{t}{variable to identify time periods}
-#'   \item{z}{policy variable}
-#'   \item{y_base}{the outcome variable}
-#'   \item{x_r}{control variable}
-#'   \item{z_fd}{first difference of z}
-#'   \item{z_fd_lead1}{1 period lead of first difference of z}
-#'   \item{z_fd_lead2}{2 period lead of first difference of z}
-#'   \item{z_fd_lead3}{3 period lead of first difference of z}
-#'   \item{z_fd_lag1}{1 period lag of first difference of z}
-#'   \item{z_fd_lag2}{2 period lag of first difference of z}
-#'   \item{z_lag3}{3 period lag of z}
-#'   \item{z_lead3}{3 period lead of z}
-#'   }
-#'
-#' @source raw dataset can be found in the .zip archive in \url{https://data.nber.org/data-appendix/w29170/}. Code for creating the example dataset can be found in `/data-raw/df_EventStudyOLS_example.R`
-"df_EventStudyOLS_example"
-
-#' Sample dataset used in the Examples section in the EventStudyFHS documentation
-#'
-#' @format A data frame with 2000 rows and 14 variables (lead and lags created using post = 1, pre = 1, overidpre = 2, overidpost = 2):
-#' \describe{
-#'   \item{id}{variable to identify units}
-#'   \item{t}{variable to identify time periods}
-#'   \item{z}{policy variable}
-#'   \item{y_base}{the outcome variable}
-#'   \item{x_r}{control variable}
-#'   \item{z_fd}{first difference of z}
-#'   \item{z_fd_lead1}{1 period lead of first difference of z}
-#'   \item{z_fd_lead2}{2 period lead of first difference of z}
-#'   \item{z_fd_lead3}{3 period lead of first difference of z}
-#'   \item{z_fd_lag1}{1 period lag of first difference of z}
-#'   \item{z_fd_lag2}{2 period lag of first difference of z}
-#'   \item{z_lag3}{3 period lag of z}
-#'   \item{z_lead3}{3 period lead of z}
-#'   \item{eta_m}{the proxy variable}
-#'   }
-#'
-#' @source raw dataset can be found in the .zip archive in \url{https://data.nber.org/data-appendix/w29170/}. Code for creating the example dataset can be found in `/data-raw/df_EventStudyFHS_example.R`
-"df_EventStudyFHS_example"
-
+"example_data"
```

Comment:
Do we need any further documentation of the variables here?

Reply:
I don't think so. In the version that was [accepted to CRAN](https://github.com/JMSLab/eventstudyr/blob/cf2365a0f5e05ac8407fd1f8ca888a48b6f3ea4e/R/data.R), we didn't describe the individual variables for `df_sample_dynamic`. Since the dataset's documentation includes a link to the paper replication files with the data generating code I think we're good as is! 

What do you think?

Reply:
Works for me thanks!